app/views/myprojects/index.html.haml

```diff
@@ -74,8 +74,8 @@
             %span #{engagement.id} (#{engagement.team_number}): #{engagement.student_names}
             %br
         %td{style: 'white-space: nowrap'}
-          - if @current_user.coach?
-            = link_to 'Edit', edit_app_path(app), :title => "Last update: #{app.updated_at.strftime('%c')}", :class => 'btn btn-primary'
-            = link_to 'Destroy', app, method: :delete, data: { confirm: 'Are you sure?' }, :class => 'btn btn-danger'
-          - if @current_user.client?
+          - if !current_user.student?

```

Comment:
Use whitelist pattern:
`- if @current_user.client? || @current_user.coach?`

Also, both clients and coaches should see only the `Request Edit` button
That is coaches should not see `Edit` or `Destroy` on the  My Projects page.